.github/workflows/build_daily.yaml

```diff
@@ -242,33 +246,11 @@ jobs:
             ${{ runner.os }}-pnpm-store-
       - name: Build artifact
         env:
-          # MACOS_CERTIFICATE: ${{ secrets.PROD_MACOS_CERTIFICATE }}
```

Comment:
These comments may be needed for subsequent front-ends, and it may be better to add them back

Reply:
okay!